webrender/src/display_list_flattener.rs

```diff
@@ -97,53 +65,24 @@ impl ClipIdToIndexMapper {
         self.add_clip_chain(id, parent_chain_index);
     }
 
-    pub fn get_clip_chain_index(&mut self, id: &ClipId) -> ClipChainIndex {
-        match self.cached_clip_chain_index {
-            Some((cached_id, cached_clip_chain_index)) if cached_id == *id =>
-                return cached_clip_chain_index,
-            _ => {}
-        }
-
-        self.clip_chain_map[id]
-    }
-
-    pub fn get_clip_chain_index_and_cache_result(&mut self, id: &ClipId) -> ClipChainIndex {
-        let index = self.get_clip_chain_index(id);
-        self.cached_clip_chain_index = Some((*id, index));
-        index
+    pub fn map_spatial_node(&mut self, id: ClipId, index: SpatialNodeIndex) {
+        debug_assert!(!self.spatial_node_map.contains_key(&id));
```

Comment:
nit: could use the returned value from `insert`

Reply:
Done